go/updater/main.go

```diff
@@ -38,12 +38,14 @@ dist-rule-directory: %s\share\suricata\rules
 }
 
 func runSuricataUpdate(installDir, dataDir, execPath string) error {
-	cmd := exec.Command(execPath,
+	args := append([]string{
 		"--suricata", filepath.Join(installDir, "bin/suricata.exe"),
 		"--config", filepath.Join(dataDir, "update.yaml"),
 		"--suricata-conf", filepath.Join(installDir, "brim-conf.yaml"),
 		"--no-test",
-		"--no-reload")
+		"--no-reload",
+	}, os.Args[1:]...)
```

Comment:
@nwt: Thanks for the suggestions to tighten this up. I had to make a couple final adjustments while incorporating in order to get it to compile and test successfully.

This first change of adding the `[1:]` was necessary because the 0th element of `os.Args[]` is the invoked command and that needed to be skipped.